README.md

```diff
@@ -29,10 +29,10 @@ in order to changes take effect. That could require a browser restart.
 NEVER set this configuration in production environments.
 
 ## Install from Source
-If you have a Go development environment set up, Go get it:
+If you have a Go development environment set up, Go get it, we recommend at least golang v1.13
```

Comment:
Maybe ". We recommend golang v1.13 or higher." instead?

Reply:
Also I believe it's always "Go" and not "golang", outside of some specific contexts (I can't find the source I saw, but you'll notice this in e.g. https://github.com/golang/go/wiki)

Reply:
yes, make sense